metadata.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
+kind: Metadata
+releaseSeries:
+- major: 1
+  minor: 5
+  contract: v1beta1
+- major: 1
+  minor: 4
+  contract: v1beta1
+- major: 1
+  minor: 3
+  contract: v1beta1
+- major: 1
+  minor: 2
+  contract: v1beta1
+- major: 1
+  minor: 1
+  contract: v1beta1
```

Comment:
@lentzi90 do you think we need to add the older contracts here? I already skipped the v1ax contracts, IMO adding older v1b1 contracts doesn't make much sense, wdyt?

Reply:
Yes I think we can safely skip them. This looks good!